src/config_check_cmd/main.go

```diff
@@ -15,21 +15,23 @@ import (
 	"github.com/envoyproxy/ratelimit/src/config"
 )
 
-func loadConfigs(allConfigs []config.RateLimitConfigToLoad) {
+func loadConfigs(allConfigs []config.RateLimitConfigToLoad, mergeDomainConfigs bool) {
 	defer func() {
 		err := recover()
 		if err != nil {
 			fmt.Printf("error loading rate limit configs: %s\n", err.(error).Error())
 			os.Exit(1)
 		}
 	}()
-	statsManager := stats.NewStatManager(gostats.NewStore(gostats.NewNullSink(), false), settings.NewSettings())
-	config.NewRateLimitConfigImpl(allConfigs, statsManager)
+	statsManager := stats.NewStatManager(gostats.NewStore(gostats.NewNullSink(), mergeDomainConfigs), settings.NewSettings())
```

Comment:
```suggestion
	statsManager := stats.NewStatManager(gostats.NewStore(gostats.NewNullSink(), false), settings.NewSettings())
```

Not sure why this line changed?